cli/src/templates/packages/react-navigation/App.tsx.ejs

```diff
@@ -54,16 +67,48 @@ export default function App() {
 
 		return (
 			<TamaguiProvider config={config}>
-				<RootStack />
+				<Navigation theme={mergedTheme} />
```

Comment:
It'd be good to move `SplashScreen.hideAsync` to the `onReady` callback:

```suggestion
				<Navigation theme={mergedTheme} onReady={() => SplashScreen.hideAsync()}  />
```

For now it'd work the same, but it's future proof if the user adds deep linking in the future.

Reply:
Will fix, thanks again 🙏

Reply:
fixed!